TwitchLib.Api.Models/Helix/Entitlements/CreateEntitlementGrantsUploadURL/CreateEntitlementGrantsUploadURLResponse.cs

```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace TwitchLib.Api.Models.Helix.Entitlements
+namespace TwitchLib.Api.Models.Helix.Entitlements.CreateEntitlementGrantsUploadURL
```

Comment:
Why are we adding an extra namespace here? and in other places. They are not needed.